mono/mini/interp/interp.c

```diff
@@ -6399,7 +6424,7 @@ call:;
 			++ip;
 			MINT_IN_BREAK;
 		MINT_IN_CASE(MINT_MONO_GET_SP)
-			sp->data.p = &frame;
+			sp->data.p = frame;
```

Comment:
Notice we get a better-seeming implementation of `GET_SP`, though it presumably does not matter.